### ISSUE TITLE
DDP training?

### ISSUE THREAD

**Issue by weimingzha0** [UofT-EcoSystem/hotline#2]
Hi,

Does it support DDP training?

**Comment by danielsnider** [comment]
No, not at the moment, but the existing algorithms in Hotline could be easily adapted to DDP. This is because DDP operates in a similar SIMD fashion to torch.DataParallel.

**Comment by Jack47** [comment]
Hey hotline is awesome, I have been always seeking this kinds of DL profiler tool~
Do you guys have plan to support DDP? I want to contribute this feature, do you have some comments about break down steps to implement DDP training support?
